api/study_api.py

```diff
@@ -19,6 +19,32 @@ def inject_html_params():
     }
 
 
+@study_api.route('/study/<string:study_id>/get_participants_api', methods=['GET'])
+def get_participants_api(study_id):
+    study = Study.objects.get(pk=study_id)
+    # `draw` is passed by DataTables. It's automatically incremented, starting with 1 on the page
+    # load, and then 2 with the next call to this API endpoint, and so on.
+    draw = int(request.args.get('draw'))
+    start = int(request.args.get('start'))
+    length = int(request.args.get('length'))

```

Comment:
Good point, but I actually think it's better for this to error if any of those fields is blank.  Otherwise we just get weird behavior that's hard to recognize or debug.  If `draw` doesn't get incremented, the table doesn't update.